tests/system/feature_store/test_feature_store.py

```diff
@@ -3130,8 +3127,41 @@ def test_onehot_with_array_values(self):
     @pytest.mark.skipif(
         not kafka_brokers, reason="MLRUN_SYSTEM_TESTS_KAFKA_BROKERS must be set"
     )
-    def test_kafka_target(self, kafka_consumer):
+    def test_kafka_target_datastore_profile(self, kafka_consumer):
```

Comment:
Just want to make sure that you ran it, since it doesn't run in CI.

Reply:
I did